README.md

```diff
@@ -5,58 +5,97 @@
 ## Table of Contents
   - [Introduction](#introduction)
   - [Prerequisite](#prerequisite)
+  - [Configuration](#configuration)
   - [Building](#building)
+  - [Testing](#testing)
   - [Contributing](#contributing)
   - [List of Contributors](#contributors)
   - [Contact](#contact)
 
 ## Introduction
 
+---
 The TrueBlocks docker version allows you to run our backend in a docker container. This will allow you to create an index of 'every appearance of every address anywhere on the chain.'
 
 With this index, your node software (i.e., [Erigon](https://github.com/ledgerwatch/erigon)) becomes an true Ethereum data server. With such a server, you can build truely distributed applications (trueDapps). TrueDapps are completely trust-less, perfectly private, use immutably data naturally, and participate without effort (by the user) in a shared, distributed application environment.
 
 ## Prerequisite
 
+---
 In order to TrueBlocks Docker to work, it must have access to an RPC endpoint with Parity tracing enabled. Erigon, works this way by default.
 
 There are commercially available offerings, but we find these inadequate because of the way TrueBlocks 'hits' the server. Erigon is easy to install, very fast syncing, and blazingly fast at serving data -- if you have an index and know what to ask for. Another great solution is running both Erigon and TrueBlocks on [dAppNode](https://github.com/dappnode) or [Avado](#).
 
 The docker build environment is also required.
 
-## Building
+## Configuration
+
+---
+We try to make Docker version as easy to configure as possible. This is why it is possible to configure both `chifra` (running inside the container) and the build.
+
+### Configuring `chifra`
+
+---
+Because `chifra` supports multiple configuration items, it is best to store them in a file. There is an example file, `env.example`, in this repository and if you choose to name your file `.env`, it will be ignored by Git.
+
+|Item|Default value|Description|
```

Comment:
This is great. Can we do two things with this?

1) Sort these items by importance -- most important first
2) Separate into two tables (if appropriate) -- one for required settings, one for optional settings.